packages/postgres-database/src/simcore_postgres_database/utils_folders.py

```diff
@@ -72,23 +77,27 @@ class FolderAccessError(FoldersError):
 
 
 class FolderNotFoundError(FolderAccessError):
-    msg_template = "no entry found for folder_id={folder_id}, gid={gid} and product_name={product_name}"
+    msg_template = "no entry found for folder_id={folder_id}, gids={gids} and product_name={product_name}"
```

Comment:
After some investigation the fronted would have received a 500. I have changed this to a 400. We also now serialise this in the reason. So yes it will be exposed directly to the fronted.

Reply:
Yes, these errors were not meant to be shown to the user. I think that the API layer above should better deal with them.